phpstan.neon

```diff
@@ -0,0 +1,5 @@
+includes:
+  - vendor/mglaman/phpstan-drupal/extension.neon
+  - vendor/mglaman/phpstan-drupal/rules.neon
+parameters:
+  customRulesetUsed: true
```

Comment:
Is this the default `phpstan.neon.dist`? If yes, do we need it? Shouldn't we directly use `phpstan.neon.dist` so that the updates are transparently handled?

Reply:
Yes, we needed it for CI/CD, but regardless, we need to run Composer for phpstan check. so dist file will be available for PHPStan. I'll remove it. However, we still need to copy phpmd.xml and phpcs.xml, even if the configuration is the same, because we require them for CI/CD as we are running checks without running Composer install.

Reply:
Yes, and if I recall, we have modified those files anyway.

Aside: is this unnecessarily confusing? Should we have the projects own the config once they install and incrementally opt-in for updated config?

Reply:
> Yes, and if I recall, we have modified those files anyway.
> 
> Aside: is this unnecessarily confusing? Should we have the projects own the config once they install and incrementally opt-in for updated config?

Yes, presently this is adding some complexity and making it confusing. Let's have projects manage their own configurations. We can develop functionality to notify users of any modifications, allowing them to review and choose whether to update it.

Reply:
@zeshanziya What do you want to do now then? Shall we just merge as-is and you'll raise another PR for all files after changes to DQC as well?

Reply:
Yes we can merge it now. Then based on our decision on issue https://github.com/axelerant/drupal-quality-checker/issues/43, we can make changes and raise another PR